draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -233,32 +233,32 @@ that the files produced can be subject to further compression using general purp
 blocking significantly reduces the CPU required to perform such strong compression. See (#simple-versus-block-coding).
 
 * Metadata about other packets received should also be included in each block. For example, counts of malformed DNS packets and non-DNS packets
-(e.g. ICMP, TCP resets) sent to the server may of interest.
+(e.g. ICMP, TCP resets) sent to the server may be of interest.
  
-It should be noted that any structured capture format that does not capture the DNS payload byte for byte will likely be limited to some extent in
-that it cannot represent "malformed" DNS packets. Only those packets that can be transformed reasonably into the structured format
-can be represented by it. So if a query is malformed this will lead to the (well formed) DNS responses with error code FORMERR appearing as "unmatched".
-
-* Data on malformed packets will optionally be recorded. There are three distinct types of packets that are considered "malformed":
-
-    * Packets that cannot be decoded into a well-formed IP or IPv6 packet, or where a valid DNS header cannot be extracted.
-      A valid DNS header is one where the identifier, flags and codes, and question count words are present and well-formed, and
-      unless the question count is 0 a single question is present in the question section.
-    * Packets with a well-formed DNS header, but well-formed records corresponding to the full count of records specified in each section
-      are not present.
-    * Packets with well-formed DNS content, but with additional data following the DNS content.
+* Data on malformed packets will optionally be recorded.
+
+    * Any structured capture format that does not capture the DNS payload byte for byte will be limited to some extent in
+      that it cannot represent "malformed" DNS packets. Only those packets that can be transformed reasonably into the
+      structured format can be represented by the format. So if a query is malformed this will lead to the (well formed) DNS responses
+      with error code FORMERR appearing as "unmatched".
+
+    * There are two distinct types of packets that are considered "malformed". These are packets which can be decoded sufficiently
```

Comment:
I think these 2 bullet points should be moved to a different section on Malformed packets. This is a design considerations section and I think this drifts into C-DSN format and implementation detail